crates/genesis/src/lib.rs

```diff
@@ -34,7 +34,7 @@ pub struct Genesis {
     #[serde(default)]
     pub config: ChainConfig,
     /// The genesis header nonce.
-    #[serde(with = "u64_hex_or_decimal")]
+    #[serde(with = "u64_hex")]
```

Comment:
these we need to keep, because the genesis format is a bit cursed and both variants are possible

Reply:
reverted the changes